src/coreclr/vm/gchelpers.cpp

```diff
@@ -49,6 +49,163 @@ inline ee_alloc_context* GetThreadEEAllocContext()
     return &t_runtime_thread_locals.alloc_context;
 }
 
+// Allocate an object on the GC heap.
+//  pEEType         -  type of the object
+//  uFlags          -  GC type flags (see gc.h GC_ALLOC_*)
+//  numElements     -  number of array elements
+//  pTransitionBlock-  transition frame to make stack crawlable
+// Returns a pointer to the object allocated or NULL on failure.
+#if defined(TARGET_X86) && defined(TARGET_UNIX)
+EXTERN_C Object* RhpGcAlloc(MethodTable* pMT, GC_ALLOC_FLAGS uFlags, uintptr_t numElements, TransitionBlock* pTransitionBlock)
+#else
+EXTERN_C Object* F_CALL_CONV RhpGcAlloc(MethodTable* pMT, GC_ALLOC_FLAGS uFlags, uintptr_t numElements, TransitionBlock* pTransitionBlock)
+#endif
```

Comment:
```suggestion
EXTERN_C Object* RhpGcAlloc(MethodTable* pMT, GC_ALLOC_FLAGS uFlags, uintptr_t numElements, TransitionBlock* pTransitionBlock)
```
Use default calling convention everywhere to avoid the ifdefs?